dozer-integrations/dozer-spring-support/dozer-spring4/src/test/java/com/github/dozermapper/spring/functional_tests/NamespaceTest.java

```diff
@@ -46,6 +48,13 @@ public void shouldRegisterMapper() {
         Mapper mapper = referencingBean.getMapper();
 
         assertNotNull(mapper);
-        assertEquals(beanMapper, mapper);
+        assertSame(beanMapper, mapper);
     }
+
+    @Test
+    public void shoulduseMappingFiles() {
```

Comment:
Minor. capitalise "use", i.e.: shouldUseMappingFiles

Reply:
I've fixed testcase method name to `shouldApplyMappingFiles`.